api-documentation.js

```diff
@@ -479,7 +591,59 @@ class ApiDocumentation extends AmfHelperMixin(LitElement) {
     }
     this.selected = e.detail.selected;
     this.selectedType = e.detail.type;
+    
+    this._updateServers();
+  }
+
+  _handleServersCountChange(e) {
+    this.serversCount = e.detail.serversCount;
   }
+
+  _updateServers() {
+    let methodId;
+    let endpointId;
+
+    if (this.selectedType === 'method') {
+      methodId = this.selected;
+    }
```

Comment:
Shouldn't here be ` else ` between the two lines? If the first condition is met then the other cannot ever be met.